packages/web/src/scss/components/UNSTABLE_EmptyState/_UNSTABLE_EmptyState.scss

```diff
@@ -26,22 +18,22 @@
     width: 100%;
 
     @include breakpoint.up(map.get(theme.$breakpoints, tablet)) {
-        flex-direction: row;
+        flex-direction: row-reverse;
     }
 }
 
-.UNSTABLE_EmptyState__wrapper {
+.UNSTABLE_EmptyState__content {
```

Comment:
Do we will have this as subcomponents? E. g. `EmptyStateContent` and `EmptyStateWrapper`?